packages/server/migrations/20230507222214_reviews.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('reviews', (table) => {
+    table.increments();
+    table.integer('movie_id').unsigned().notNullable();
+    table.foreign('movie_id').references('id').inTable('movies');
+    table.integer('user_id').unsigned().notNullable();
+    table.foreign('user_id').references('id').inTable('users');
+    table.integer('rating').notNullable();
```

Comment:
Let's try with enum, as the schema says 🙏🏽 
I know I did not to it for Categories and perhaps did not give a good example 🙈 

https://knexjs.org/guide/schema-builder.html#enum-enu:~:text=enum%20/%20enu-,%23,-table.enu(col

Reply:
Hi,
Thanks for reviewing my work.
I have made necessary changes in the file.
Please review it.